docs/src/layouts/Main.astro

```diff
@@ -22,6 +23,7 @@ if (currentPage) {
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     
     <title>{content.title ? `${content.title} 🚀 ${site.title}` : site.title}</title>
+    <MetaData {content} canonicalURL={Astro?.request?.canonicalURL}/>
```

Comment:
it would be nice to remove some of these "unknown" optional chaining `?` symbols if we can. Does any page not currently have `canonicalURL`?

Reply:
I don't know, I haven't use the `Astro.request` api enough to know, I guess. I might as well remove those checks and the build can just fail instead.